interpreter/syntax/types.ml

```diff
@@ -11,11 +11,13 @@ type mutability = Immutable | Mutable
 type table_type = TableType of Int32.t limits * ref_type
 type memory_type = MemoryType of Int32.t limits
 type global_type = GlobalType of value_type * mutability
+type exn_type = func_type
```

Comment:
I suppose we'll need a richer type structure later on, but at the moment I think `func_type` suffices.